var/spack/repos/builtin/packages/py-pyfftw/package.py

```diff
@@ -13,21 +13,28 @@ class PyPyfftw(PythonPackage):
     homepage = "http://hgomersall.github.com/pyFFTW"
```

Comment:
Dead link. Unofficial link.
```suggestion
    homepage = "https://pyfftw.readthedocs.io/en/latest/"
```

Reply:
Has been added to the PR, done.